### DISCUSSION TITLE
When I upload a recording file to s3, can I specify it as a different folder??

### DISCUSSION THREAD

**Discussion by Jamong435** [ant-media/Ant-Media-Server#5957] [Q&A]
hello~~
I uploaded it after recording and confirmed it on s3
However, I would like to make separate folders to check who uploads it
Is it possible?

Or can I get the file name that goes up to s3 as the return value?

thanks!

**Comment by yashtandon113** [comment]
@Jamong435
Yes, it is possible. Please check out this [GitHub discussion](https://github.com/orgs/ant-media/discussions/5122) to use subfolder for the streams, and it works for the S3 bucket as well.

Generally, as per the naming structure, the VOD file name is `streamId.mp4`, so the same will be uploaded to the S3 bucket.

Please let us know if it helps.
